nuget/lib/dependabot/nuget/file_fetcher.rb

```diff
@@ -24,13 +24,14 @@ def self.required_files_in?(filenames)
         return true if filenames.any? { |f| f.end_with?(".sln") }
         return true if filenames.any? { |f| f.match?("^src$") }
         return true if filenames.any? { |f| f.end_with?(".proj") }
+        return true if filenames.any? { |f| f.match?(/^dotnet\-tools\.json$/i) }
```

Comment:
Not being a ruby person, I'm not sure what the best way to refactor this to make the linter happy about the perceived complexity.

Reply:
This change may allow the file_fetcher to run but it doesn't change how the updater works and it currently is expecting a project file to work against.

Reply:
Could you point me at the rough area of code I need to go and touch? I started off by searching for the error message I got in the GitHub UI.

Reply:
May have to hold off for a bit as we are trying to move more of the updater logic from ruby to C# for the NuGet ecosystem. The new `discover` command in the NuGetUpdater.Core project is now responsible for parsing dependencies. It currently uses project files as its starting point but could be updated to simply look for global.json and .config/dotnet-tools.json in the target directory. The `update` command in NuGetUpdater.Core is responsible for performing the update and will be going through some changes this month.